common/environment/build-style/void-cross.sh

```diff
@@ -0,0 +1,12 @@
+lib32disabled=yes
+nocross=yes
+nopie=yes
+create_wrksrc=yes
+
+if [ "$cross_libc" = "glibc" -a "$XBPS_TARGET_LIBC" != "glibc" ]; then
+    broken="glibc crosstoolchains are only available on glibc hosts for now"
+fi
+
+if [ "cross_wordsize" = "64" -a "$XBPS_TARGET_WORDSIZE" != "64" ]; then
+    broken="64-bit crosstoolchains are not available on 32-bit hosts"
+fi
```

Comment:
As you've noted, cross toolchain for 64 bit platforms on 32 bit platform should work. It's only waiting for someone(TM) to test and enable it. But, I prefer to put them in each template, to allow enabling those packages case by case.
I visited cross toolchain for x86_64 on i686 the other days, for grub-efi, it works there. IMO, let's spell the broken in each template.

Reply:
imo we don't have to put broken= anywhere, let's just enable it

if builders fail, we can put broken= in there after the fact